docs/rfcs/016-holdings_service/README.md

```diff
@@ -0,0 +1,128 @@
+# **RFC 016: Holdings service**
+
+**Last updated: 23 September 2019.**
+
+## **Background**
+
+As we move a greater proportion of people over from wellcomelibrary.org to
+wellcomecollection.org we need to provide a service allowing people to query on
+and request items based on their availability and
+location, whether it be online, in the library, or something else.
+
+This document will outline the different services we will need build and
+integrate with.
+
+We will explore the best way to start developing the service and get it to
+people for research and testing as quickly as possible, without losing sight of
+the fact that this service will need to integrate with Sierra, CAS and other
+services while containing other integral pieces of infrastructure.
+
+We will also assume that we will not be disrupting any current processes across
+teams outside of the digital platform, or adding more places for library members
+to manage their relationship with us (i.e. account information).
+
+The project will most likely be a cross team (experience / data / preservation)
+endeavour, so the process in how we communicate and share what we are doing will
+be important too.
+
+## **Proposed solution**
+
+[https://docs.google.com/drawings/d/1jAAiUS2A2wifiV2ZL5RkdPzHDHvOzTSQPE-HWwxeyMU/edit?usp=sharing](https://docs.google.com/drawings/d/1jAAiUS2A2wifiV2ZL5RkdPzHDHvOzTSQPE-HWwxeyMU/edit?usp=sharing)
```

Comment:
Can we keep these self contained please - include the image locally in the repo.

Reply:
Sure thing, I'll update it once we get some of the more fundamental questions answered (like the one on catalogue API integration) - otherwise I am just doing loads of export PNG stuff.

Reply:
We'll need to get a shared understanding of some of the language: _holding_, _availability_

Reply:
What language / definitions are we currently using? Or is that still TBD?

Reply:
Glossary section in the RFC?

Reply:
> We'll need to get a shared understanding of some of the language: _holding_, _availability_

You know who'd be really good at defining this? ping @pollecuttn 

Can we have a glossary pls?